docs/references/ayx-plugin-cli.md

```diff
@@ -0,0 +1,487 @@
+					"input_anchors": {
+						"Input": {
+							"label": "",
+							"allow_multiple": false,
+							"optional": false
+						}
+					},
+					"output_anchors": {
+						"Output": {
+							"label": "",
+							"allow_multiple": false,
+							"optional": false
+						}
+					}
+				}
+			}
+		},
+		"tool_version": "1.0"
+	}
+
+
+
+# AYX Plugin CLI Overview
+
+### CLI Architecture
+
+The AYX Plugin CLI is a command-line interface based on
+[Typer](https://typer.tiangolo.com/),
+a tool that helps you build and manage custom plugins for Alteryx
+Designer. This top-level CLI oversees the creation and metadata of your
+plugins. It delegates the tasks of creating and building YXIs.
+
+The AYX Plugin CLI detects which SDK your workspace is using to develop
+plugins and delegates the other CLI commands to their respective SDKs.
+For example, if you are developing Python tools, the CLI calls the
+Python implementation of `create-ayx-plugin`. However, if you are using
+another SDK to develop tools, the CLI delegates the `create-ayx-plugin`
+call to that SDK's implementation.
+
+This architecture allows for one CLI to manage the entire workspace,
+abstracting away the details of the commands to their respective SDKs.
+This lets you develop in different SDK languages without the need to
+download multiple CLI tools.
+
+### Exploring the AYX Plugin CLI
+
+You can use the command line to access information on CLI usage, as well
+as available commands and their descriptions.
+
+-   **AYX Plugin CLI**: Use the `ayx_plugin_cli` command (with no
+    arguments) to open help content for the AYX Plugin CLI (top-level
+    CLI).
+-   **Python SDK CLI**: Use the `ayx_python_sdk` command (with no
+    arguments) to open help content for the Python SDK CLI.
+
+You can also pass in the `--help` argument to display the same
+information as described above.
+
+The help menu contains 3 sections:
+
+-   **Usage**: The Usage section describes the order in which commands
+    and options are passed.
+-   **Options**: The Options section lists the available options and
+    their descriptions.
+-   **Commands**: The Commands section lists the available commands and
+    their descriptions.
+
+### AYX Plugin CLI General Usage
+
+To create a default tool with the AYX Plugin CLI, follow these steps.
+
+#### Initialize Workspace
+
+To initialize the plugin workspace, run `sdk-workspace-init`. You will
+be asked to define these items:
+
+-   Package Name: The name of the folder where you will create tools.
+-   Tool Category: The Alteryx Designer tool category where all of the
+    tools in this workspace will appear.
+-   Description: This information is presented to the user when they
+    install a YXI.
+-   Author: Who wrote the plugins?
+-   Company: What company owns these plugins?
+-   Back-end Language: Currently, Python is the only option.
+
+#### Add Plugin
+
+Once your workspace is ready, run `create-ayx-plugin`. You will be asked
+to define these items:
```

Comment:
TW review:

Once your workspace is ready, run `create-ayx-plugin`. You need
to define these items:

Reply:
fixed